google/ggrc-core
# QUICK-FIX Forbid /admin/full_reindex and /admin/reindex_snapshots

# Issue description

`/admin/full_reindex` and `/admin/reindex_snapshots` endpoints should be deprecated now in favor of `/admin/reindex` since they don't work properly.

# Steps to test the changes

1. Run GGRC app and login as Superuser;
2. Try to perform `POST `/admin/full_reindex` and look in devtools for response;
3. Try to perform `POST `/admin/reindex_snapshots` and look in devtools for response.

Expected result: `405` status code should be returned with appropriate error response telling that this endpoints are deprecated in favor of `/admin/reindex`.

# Solution description

This solutions includes:
* New decorator `deprecatted_endpoint` which could be used to mark an endpoint as a deprecated one. New endpoint could be provided as a parameter here.`

* Endpoints starting background tasks for full reindex and snapshots reindex are decorated with `deprecatted_endpoint`.

# Sanity checklist




# PR Review checklist
